static/js/main.js

```diff
@@ -93,16 +99,19 @@ firebase.auth().signInWithPopup(provider).then(() => {
           db.child('cases').child(CURRENT_CASE).child('messages')
             .once('value', (s) => {
               $('.messageSpace').html(''); // wipe messageSpace content
-              s.val().forEach((message) => {
-                if (message.sender === 'user') {
-                  $('.messageSpace')
-                    .append(`<div class="message from" id="${message}">${message.body}</div>`);
-                } else {
-                  $('.messageSpace')
-                    .append(`<div class="message to" id="${message}">`
-                          + `${s.val()[message].body}</div>`);
-                }
-              });
+              if (s.val() !== null) {
+                console.log(s.val());
```

Comment:
If the Firebase message object is empty, then the code breaks because it doesn't know how to iterate over an empty object.